CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# Samply.Beam 0.4.1 -- 2022-12-08
+
+This is a bugfix release. In particular, the following things were addressed:
+
+* An error, where only the first result from multiple was decrypted is fixed.
+* Due to computer time mismatches, sometimes JWTs "from the future" were rejected. The Broker now accepts those tokens. Note, that expired tokens are still rejected.
+* The Broker explicitly rejects unencrypted messages. This is the same behaviour as in version 0.4.0, however the guarantees to never accept malformed messages with unencrypted payloads are strengthened.
```

Comment:
@TKussel I don't think that's true. I tried putting that in via a serde annotation but failed and reverted the commit. Can't see such a change in the code.

I suggest leaving the code as-is but removing this bullet point.

Reply:
You are right, my eyes skipped over commit 4dd7edf01ffa631bf656e52b06f8dc4e62933d74, maybe I was too trained on the term “revert”. I will fix the docs promptly.